panel/src/components/Forms/Blocks/BlockSelector.vue

```diff
@@ -114,10 +114,7 @@ export default {
     },
     navigate(index) {
       const ref = this.$refs["fieldset-" + index];
-
-      if (ref?.[0]) {
-        ref[0].focus();
-      }
+      ref?.[0].focus?.();
```

Comment:
Isn't this wrong? Question is not whether `ref` exists, but if it's empty or has actually a first element

Reply:
Absolutely! Great catch. Should be:

```js
ref?.[0]?.focus?.();
```

Reply:
As for other parts of the code: the optional call on `focus? is probably redundant. However, as long as we don't use TypeScript we'd better be safe...